hook!-full-game/src/Autoload/Utils.gd

```diff
@@ -17,3 +17,12 @@ static func get_aim_joystick_strength() -> Vector2:
 	return Vector2(
 		Input.get_action_strength("aim_right") - Input.get_action_strength("aim_left"),
 		Input.get_action_strength("aim_down") - Input.get_action_strength("aim_up"))
+
+"""
+Checks if two numbers are approximately equal
+"""
+static func is_equal_approx(a: float, b: float, cmp_epsilon: float = 1e-5) -> bool:
+	var tolerance: = cmp_epsilon * abs(a)
+	if tolerance < cmp_epsilon:
+		tolerance = cmp_epsilon
+	return abs(a - b) < tolerance
```

Comment:
You should ensure that there are newlines at the end of each file. Godot will begin enforcing this in 3.2, it's the POSIX standard, and GitHub shows a red error symbol otherwise.

Reply:
I use a script to clean up the project from time to time (e.g. when we release a new version), but that should be automated - too easy for a dev to forget this. Many devs aren't even aware of this or that it can break cli tools.